cli/main.rs

```diff
@@ -295,15 +304,23 @@ async fn compile_command(
   flags: Flags,
   source_file: String,
   output: Option<PathBuf>,
+  args: Vec<String>,
 ) -> Result<(), AnyError> {
   if !flags.unstable {
     exit_unstable("compile");
   }
 
   let debug = flags.log_level == Some(log::Level::Debug);
 
+  let preload_flags = Flags {
+    inspect: None,
+    inspect_brk: None,
```

Comment:
Maybe `inspect` and `inspect_brk` shouldn't be available to `compile` subcommand at all?

Reply:
> Maybe `inspect` and `inspect_brk` shouldn't be available to `compile` subcommand at all?

I included them because it seems simpler to group them into the wider set of "flags that affect runtime behaviour", and match `deno install` in doing so. On the other hand, is there potential to remove the inspector from the light runtime binary?

Reply:
> On the other hand, is there potential to remove the inspector from the light runtime binary?

We can feature gate it on our side; it will break features like the REPL but those seem irrelevant for compiled bundles.

Reply:
> On the other hand, is there potential to remove the inspector from the light runtime binary?

Not in the near future, no.

> We can feature gate it on our side; it will break features like the REPL but those seem irrelevant for compiled bundles.

REPL is not part of `runtime/` directory, it's `cli/` only so that should work fine

Reply:
Doh, apparently I already went out of my way to exclude the inspector flags in `flags.rs` 🤦‍♂️ I'll just remove this meaningless handling.